### DISCUSSION TITLE
v1和v2路由可以暂时共存吗？

### DISCUSSION THREAD

**Discussion by dzx-dzx** [DIYgod/RSSHub#12750] [Q&A] [answered]
我在v2规范下修复了https://github.com/DIYgod/RSSHub/issues/12742 ，但不太有空迁移路由下其它的几个路径。有可能让他们共存一段时间，或者将修改应用在v1上吗，还是必须v2迁移完后才能提交？

**Comment by TonyRL** [comment, marked as answer]
They can coexist as long as the path isn't the same. You can take a look at `/gov/...` in [v1](https://github.com/DIYgod/RSSHub/blob/master/lib/router.js) and [v2](https://github.com/DIYgod/RSSHub/blob/master/lib/v2/gov/router.js) router.

**Reply by dzx-dzx** [reply]
Thanks? And it is also OK to make such a PR?
